### PR TITLE
Sittercity technology internship blog post

### PR DESCRIPTION
**Pull request by ellie007**

### CODE REVIEW

**Comment by ppeble** [comment]
@ellie007 Looks great! My only suggestion is to make actual headers out of your sections. For example, on line 17 it seems like a natural place to make a new section header.

Thanks for contributing! We'll hopefully be putting this up within the next week or so.

**Comment by ellie007** [comment]
@ptrimble - I added the headers; Please let me know if anything else!

**Comment by ppeble** [comment]
Looks great! I think we're good. I handle merging all of this in once we get the go-ahead from Sam. Thanks!

File: _posts/Sittercity Technology Internship! [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "Sittercity Technology Internship!"
+date:   2015-03-19 09:26:00
+categories: technology
+comments: true
```
Comment (ppeble) [inline _posts/Sittercity Technology Internship!]: We need to add Eleanor as the author here.

**Comment by ellie007** [comment]
Hehe, I am still getting these notifications and it said it needs my name added as the author so I thought I would just do it; Done!

**Comment by ppeble** [comment]
Oh thanks! Appreciate it. :smile:

**Pull request merged**